plugins/spu2-x/src/Windows/Spu2-X.rc

```diff
@@ -24,24 +24,6 @@ LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
 // Dialog
 //
 
-IDD_ABOUT DIALOGEX 0, 0, 291, 223
-STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "About SPU2-X"
-FONT 8, "MS Shell Dlg", 400, 0, 0x1
-BEGIN
-    DEFPUSHBUTTON   "I've Seen Enough",IDOK,91,204,111,14
-    CTEXT           "Original portions from SPU2ghz are (c) 2008 by David Quintana [gigaherz]",IDC_STATIC,9,184,273,11
-    CONTROL         114,IDC_STATIC,"Static",SS_BITMAP,45,5,203,64,WS_EX_CLIENTEDGE
```

Comment:
Note that 114 corresponds to the IDB_SPU2X bitmap, and that it isn't used on any other dialog.

Reply:
Done.